src/common/Footer.tsx

```diff
@@ -0,0 +1,58 @@
+import { useNavigate } from 'react-router-dom';
+import styled from 'styled-components';
+import { IcFooterEmail, IcFooterInsta, IcFooterLogo } from '../assets';
+import { handleClickLink } from '../utils/handleClickLink';
+
+const Footer = () => {
+  const navigate = useNavigate();
+
+  return (
+    <FooterContainer>
+      <FooterInfoContaniner>
+        <div onClick={() => navigate('/')}>
+          <IcFooterLogo />
+        </div>
```

Comment:
P2: onClick 함수 추가만을 위해서라면 div 태그를 하나 더 추가하지 않아도 대요!
```suggestion
   <IcFooterLogo onClick={() => navigate('/')} />
```

Reply:
아하! 요부분 수정하겠습니당